src/commands/countdown/subcommands/list.ts

```diff
@@ -0,0 +1,62 @@
+import dayjs from 'dayjs';
+import * as Discord from '../../../Discord';
+import SubCommand from '../../../SubCommand';
+import { bold, fancyCenter } from '../../../format';
+
+type MonthGroup = {
+  month: string;
+  year: number;
+  heading: string;
+  list: string;
+};
+
+const list = new SubCommand({
+  name: 'list',
+  displayName: 'List Countdowns',
+  description: 'List all stored countdowns',
+  handler: async ctx => {
+    const guildId = ctx.mustGetGuildId();
+    const dates = await (await ctx.countdowns().scan(guildId))
```

Comment:
Is the double `await` necessary?

Reply:
Yeah I think so, as `toArray()` returns a `Promise`.